haiku-apps/documentviewer/documentviewer-0.3~git.recipe

```diff
@@ -22,7 +22,7 @@ REQUIRES="
 	lib:libfreetype$secondaryArchSuffix
 	lib:libjbig2dec$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
-	lib:libmupdf$secondaryArchSuffix
+	lib:libmupdf$secondaryArchSuffix >= 1.20
```

Comment:
This one should be automatically set by haikuporter

Reply:
Okay, done in latest commit, thanks